example-app/dependency-reduced-pom.xml

```diff
@@ -3,7 +3,7 @@
   <parent>
     <artifactId>dw-consul</artifactId>
     <groupId>com.boundary.dropwizard.consul</groupId>
-    <version>0.16-SNAPSHOT</version>
+    <version>0.18-SNAPSHOT</version>

```

Comment:
actually this file got into source by accident - it shouldn't be included at all and `dependency-reduced-pom.xml` should be in .gitignore. Can you go ahead and delete it?